### ISSUE TITLE
LM strategy - Add minimum spread parameter to config

### ISSUE THREAD

**Issue by dennisocana** [hummingbot/hummingbot#3034]
Creating a separate story as per some requests from the community and in this comment [here](https://github.com/CoinAlpha/hummingbot/issues/2932#issuecomment-790646802).

![image.png](https://images.zenhubusercontent.com/5f597e9f5e2dba41ca21ad9d/a28fac7d-9e3a-4f79-b275-f14bd3a113f8)

┆Issue is synchronized with this [Clickup task](https://app.clickup.com/t/1ram8rr) by [Unito](https://www.unito.io)

**Comment by MementoRC** [comment]
@rapcmia Is this not implemented?

**Comment by rapcmia** [comment]
@MementoRC yes correct. At the moment we dont have any plans of improvement for the strategy, else community can discuss and fund a bounty for the feature
